src/Ucommerce.Avenue.Umbraco/Controllers/CategoryController.cs

```diff
@@ -1,49 +1,46 @@
 ﻿using System.Collections.Generic;
 using System.Web.Mvc;
-using UCommerce.Api;
+using Ucommerce.Api;
+using Ucommerce.Api.Search;
 using UCommerce.Content;
-using UCommerce.EntitiesV2;
 using Umbraco.Web.Mvc;
-using UCommerce.Extensions;
 using UCommerce.Infrastructure;
 using UCommerce.RazorStore.Models;
-using UCommerce.Runtime;
-using UCommerce.Search.Facets;
+using UCommerce.Search;
+using UCommerce.Search.FacetsV2;
+using UCommerce.Search.Models;
 using Umbraco.Web.Models;
 
 namespace UCommerce.RazorStore.Controllers
 {
     public class CategoryController : RenderMvcController
     {
+        private readonly ICatalogLibrary _catalogLibrary = ObjectFactory.Instance.Resolve<ICatalogLibrary>();
+        private readonly ISiteContext _siteContext = ObjectFactory.Instance.Resolve<ISiteContext>();
+        private readonly ISearchLibrary _searchLibrary = ObjectFactory.Instance.Resolve<ISearchLibrary>();
+
         public override ActionResult Index(ContentModel model)
         {
-            var currentCategory = SiteContext.Current.CatalogContext.CurrentCategory;
+            var currentCategory = _siteContext.CatalogContext.CurrentCategory;
 
             var categoryViewModel = new CategoryViewModel
             {
-                Name = currentCategory.DisplayName(),
-                Description = currentCategory.DisplayName(),
-                CatalogId = currentCategory.ProductCatalog.Id,
-                CategoryId = currentCategory.Id,
+                Name = currentCategory.DisplayName,
+                Description = currentCategory.DisplayName,
```

Comment:
Make it `Description = currentCategory.Description` instead, the same nonsense was present in **ProductController as well**.

Reply:
yep, changed it